pkg/ibmvpc/nacl_analysis.go

```diff
@@ -378,31 +380,70 @@ func (na *NACLAnalyzer) addAnalysisPerSubnet(subnetCidr string) {
 
 	fmt.Printf("\naddAnalysisPerSubnet results:\n")
 	fmt.Printf("subnetCidr: %s\n", subnetCidr)
-	fmt.Printf("%s", na.GeneralConnectivityPerSubnet(subnetCidr))
+	subnetConnectivityResStr, _ := na.GeneralConnectivityPerSubnet(subnetCidr)
+	fmt.Printf("%s", subnetConnectivityResStr)
 
 	fmt.Println("-----")
 }
 
 func getDisjointSubnetCidrStr(disjointSubnetCidr string) string {
-	return fmt.Sprintf("local ip-block: %s\n", disjointSubnetCidr)
+	return fmt.Sprintf("\nlocal ip-block: %s\n", disjointSubnetCidr)
+}
+
+// currently assuming only subnet-level connectivity result is required
+// TODO: support refinement to partial subnet level when required
+/*type SubnetConnectivityResult struct {
+	allowedIngressConns map[*common.IPBlock]*common.ConnectionSet
+	allowedEgressConns  map[*common.IPBlock]*common.ConnectionSet
+}*/
+
+func getConnectivityResultMapString(m map[string]*ConnectivityResult) string {
+	strResult := ""
+	sortedKeys := []string{}
+	for k := range m {
+		sortedKeys = append(sortedKeys, k)
+	}
+	sort.Strings(sortedKeys)
+
+	for _, k := range sortedKeys {
+		disjointSubnetCidr := k
+		connectivityRes := m[k]
+		strResult += getDisjointSubnetCidrStr(disjointSubnetCidr)
+		strResult += connectivityRes.string()
+	}
```

Comment:
```suggestion
	for _, k := range sortedKeys {
		connectivityRes := m[k]
		strResult += getDisjointSubnetCidrStr(k)
		strResult += connectivityRes.string()
	}
```

Reply:
changed this function in the next PR (branch connectivity_grouping_issue_74 on top of this branch).